Cargo.toml

```diff
@@ -9,23 +9,28 @@ license-file = "LICENCE"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dev-dependencies]
-criterion = "0.3.5"
+criterion = "0.5.1"
 quickcheck = "1.0.3"
 
 [dependencies]
-aes = { version = "0.8.2", features = ["zeroize"]}
-block-modes = "0.8.1"
+aes = { version = "0.8.2", features = ["zeroize"] }
+block-modes = "0.9.1"
 byteorder = "1.4.3"
-hex-literal = "0.3.2"
+hex-literal = "0.4.1"
 rand = "0.8.5"
 rand_chacha = "0.3.1"
 num = "0.4.0"
 hex = "0.4.3"
 subtle-ng = "2.5.0"
-zeroize = { version = "1.5.7", features = [ "zeroize_derive", "alloc" ] }
+zeroize = { version = "1.5.7", features = ["zeroize_derive"] }
 lazy_static = "1.4.0"
 thiserror = "1.0.38"
 
+[patch.crates-io]
+# version 3.16.0
+bumpalo = { git = 'https://github.com/fitzgen/bumpalo.git', rev = "4eeab88" }
```

Comment:
Do we have to use git deps? can't this just be `bumpalo = "3.16.0"`?

Reply:
For some reason, can only patch by providing a different `source` for the crate. 
Rather annoying.